src/components/Signup/Signup.js

```diff
@@ -2,26 +2,26 @@ import React, { Component, PropTypes } from 'react';
 import { Link } from 'react-router';
 
 import '../../styles/Signup.scss';
+import '../../styles/Common.scss';
 import SignupForm from './SignupForm';
 
 class Signup extends Component {
   render() {
     const { submitSignup, signupError } = this.props;
 
     return (
-      <div>
-        <div>
-          <h2>Signup</h2>
-          <SignupForm
-            submitSignup={submitSignup}
-          />
-          {
-            signupError ?
-            <span>{signupError}</span> :
-            null
-          }
-          <Link to="/">Log In</Link>
-        </div>
+      <div className="container">
+        <h1 className="title">SIGN UP</h1>
+        <SignupForm
+          submitSignup={submitSignup}
+        />
+        {
+          signupError ?
```

Comment:
@juanlacruz You might find code like this one on the repo, I learned that better syntax @bernabe9 suggested after creating this, so if you stumble upon one of these, change it. Also, if you feel inspired, submit a PR improving the base ;)